yugabyte/src/yugabyte/core.clj

```diff
@@ -2,50 +2,90 @@
   "Integrates workloads, nemeses, and automation to construct test maps."
   (:require [clojure.tools.logging :refer :all]
             [clojure.string :as str]
-            [jepsen [checker :as checker]
-                    [generator :as gen]
-                    [tests :as tests]]
+            [clojure.pprint :refer [pprint]]
+            [jepsen.checker :as checker]
+            [jepsen.client :as client]
+            [jepsen.generator :as gen]
+            [jepsen.tests :as tests]
             [jepsen.control.util :as cu]
-            [jepsen.os [debian :as debian]
-                       [centos :as centos]]
-            [yugabyte [auto :as auto]
-                      [bank :as bank]
-                      [counter :as counter]
-                      [long-fork :as long-fork]
-                      [multi-key-acid :as multi-key-acid]
-                      [nemesis :as nemesis]
-                      [single-key-acid :as single-key-acid]
-                      [set :as set]]))
+            [jepsen.os.debian :as debian]
+            [jepsen.os.centos :as centos]
+            [yugabyte.auto :as auto]
+            [yugabyte.bank :as bank]
+            [yugabyte.counter :as counter]
+            [yugabyte.long-fork :as long-fork]
+            [yugabyte.multi-key-acid :as multi-key-acid]
+            [yugabyte.nemesis :as nemesis]
+            [yugabyte.single-key-acid :as single-key-acid]
+            [yugabyte.set :as set]
+            [yugabyte.utils :refer :all]
+            [yugabyte.ycql.bank]
+            [yugabyte.ycql.counter]
+            [yugabyte.ycql.long-fork]
+            [yugabyte.ycql.multi-key-acid]
+            [yugabyte.ycql.set]
+            [yugabyte.ycql.single-key-acid]
+            [yugabyte.ysql.counter])
```

Comment:
As I remember auto-formatting was also having some other issues and there is no need to use it for the whole files. We can just don't use auto-formatting for imports and keep them as is.

Reply:
Auto-formatting works fine for most files. We could've left imports as they were, but should we really? I'd argue that unrolled imports are better for readability and are easier to reorder.

Reply:
I think it is better to stick to Jepsen project code style because eventually our tests will be merged into parent repository and we would like to avoid spending time twice on re-formatting first as we want and then re-formatting as required by Jepsen on merging back.

Reply:
Jepsen itself doesn't seem to have a hard stance on the import formatting, see e.g.
https://github.com/jepsen-io/jepsen/blob/master/consul/src/jepsen/consul.clj#L2-L15
where imports aren't combined, or
https://github.com/jepsen-io/jepsen/blob/master/faunadb/src/jepsen/faunadb/auto.clj#L4-L20
where both styles are used